src/components/JobOffers/JobOffers.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './JobOffers.scss'

```

Comment:
**nitpick:** When we work with react components we should not use global styles, as it can lead to style overwriting when your app grows. You can use here CSS modules outside the box which isolates styles from each other, and then you don't have to worry about CSS naming convention like BEM. Check out this article for quick intro: https://blog.openreplay.com/using-css-modules-in-react/